library/core/src/num/nonzero.rs

```diff
@@ -513,6 +513,33 @@ nonzero_unsigned_operations! {
 macro_rules! nonzero_signed_operations {
     ( $( $Ty: ident($Int: ty) -> $Uty: ident($Uint: ty); )+ ) => {
         $(
+            #[stable(feature = "nonzero_checked_ops", since = "1.63.0")]
+            impl Neg for $Ty {
+                type Output = Self;
+                /// Negate the non-zero value.
```

Comment:
suggestion: include a paragraph here about this behaving like `iN::neg` regarding overflow -- `-(NonZeroI8::new(-128)?)` is `-128` in release mode, since that's still non-zero.  I think all the other ops that can overflow are omitted because they *can* result in zero, so people might incorrectly extrapolate that `Neg` doesn't overflow here.

(Personally I might say this should be a separate PR, because it's not really a "partial stabilization" diff, but it's also unambiguously fine, IMHO, so it doesn't really bother me either way.)

Reply:
Okay, I'll clarify this :)
By the time I came across your comment, the commit was already moved to https://github.com/rust-lang/rust/pull/97547, so I guess we do have a separate PR now ^ ^".